ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -71,6 +67,23 @@ def get_node_port_display(self, port: Port) -> PortDisplay:
     def get_from_node_display_registry(cls, node_class: Type[NodeType]) -> Type["BaseNodeDisplay"]:
         return cls._node_display_registry[node_class]
 
+    @classmethod
+    def infer_node_class(cls) -> Type[NodeType]:
```

Comment:
[q] Does this work as expected for adorned nodes?

Reply:
What would be you expectation?

Reply:
Not sure tbh. I suppose that it'd return `TryNode`

Reply:
yea right now it's meant to simply return the node class that is parameterized